### PR TITLE
Cumulus 1556 add deployment

### PR DESCRIPTION
**Pull request by Jkovarik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,5 @@
...
-```bash
-npm install @cumulus/<package-name>@^1.11.3 --save
-```
-
-**Note**: Using `^1.11.3` instead of `1.11.3` means that when you run
-`npm install` for this project in the future, if there are any newly released
-minor versions of that package (e.g. `1.11.4` or `1.12.0`), they will be
-installed automatically. However, if you want to stay pinned to version `1.11.3`
-for the future, you can install using that exact version.
+[https://nasa.github.io/cumulus/docs/deployment/deployment-readme](https://nasa.github.io/cumulus/docs/deployment/deployment-readme)
```
Comment (Jkovarik) [inline README.md]: Docs are somewhat dependent on the doc story.

File: data-persistence-tf/main.tf [hunk 2]
```diff
@@ -0,0 +1,50 @@
+module "data_persistence" {
```
Comment (markdboyd) [inline data-persistence-tf/main.tf]: Can we split the contents of the file out into `main.tf`, `variables.tf`, and `outputs.tf`, as we do for `cumulus-tf`?

Comment (Jkovarik) [reply data-persistence-tf/main.tf]: I suppose we *could*.  It's not what we're doing in `/example` currently, and I'm not sure I have a good handle on a best practice here for small modules.

Comment (markdboyd) [reply data-persistence-tf/main.tf]: We identify it as a best practice in [`components.md`](https://github.com/nasa/cumulus/blob/CUMULUS-1560-deployment-docs/docs/deployment/components.md). 

I also think it's kinda confusing to see variables defined in the same place as the use of the module, but it's debatable

File: cumulus-tf/main.tf [hunk 3]
```diff
@@ -1,8 +1,12 @@
 module "cumulus" {
-  source = "https://github.com/nasa/cumulus/releases/download/{changeme}/terraform-aws-cumulus.zip//tf-modules/cumulus"
+  source = "https://github.com/jkovarik/cumulus/releases/download/v1.27.0/terraform-aws-cumulus.zip//tf-modules/cumulus"
 
   cumulus_message_adapter_lambda_layer_arn = var.cumulus_message_adapter_lambda_layer_arn
 
+  archive_api_port            = var.archive_api_port
+  private_archive_api_gateway = var.private_archive_api_gateway
```
Comment (nemreid) [inline cumulus-tf/main.tf]: These two are duplicated at the bottom of this module definition.

File: cumulus-tf/main.tf [hunk 4]
```diff
@@ -0,0 +1,91 @@
+module "cumulus" {
+  source = "https://github.com/jkovarik/cumulus/releases/download/v1.27.0/terraform-aws-cumulus.zip//tf-modules/cumulus"
+
+  cumulus_message_adapter_lambda_layer_arn = var.cumulus_message_adapter_lambda_layer_arn
+
+  archive_api_port            = var.archive_api_port
+  private_archive_api_gateway = var.private_archive_api_gateway
```
Comment (nemreid) [inline cumulus-tf/main.tf]: These two are duplicated at the bottom of this module definition.

**Pull request merged**